packages/neuron-wallet/src/locales/en.ts

```diff
@@ -215,6 +215,12 @@ export default {
           ok: 'OK',
           cancel: 'Cancel'
         }
+      },
+      'node-version-different': {
+        message: '节点版本与Neuron不一致，请确认后使用'
```

Comment:
```suggestion
        message: '节点版本与 Neuron 不一致，请确认后使用'
```

What if the desired version is prompted in the message, e.g. `Neuron (CKB Node@v0.108.1)`

Reply:
Do you mean we need to display the internal node version and the external node version?

Reply:
I think showing versions in comparison would be more friendly and easy to debug with a screenshot, any idea from @Danie0918

Reply:
This will be better and help users understand the current wallet version.Supplementary instructions can be added as follows：

[英]The node version is inconsistent with Neuron(v 0.108.1), please use after confirmation

[中]节点版本与Neuron(v 0.108.1)不一致，请确认后使用